pennylane/qchem/openfermion_pyscf.py

```diff
@@ -913,7 +914,9 @@ def meanfield(
         )
         raise TypeError(error_message)
 
-    filename = name + "_" + package.lower() + "_" + basis.strip()
+    ct = datetime.datetime.now()
+    filename = name + "_" + package.lower() + "_" + basis.strip() + "_" + str(ct.timestamp())
+
```

Comment:
How does this new name structure affect the packages created? If existing packages expect the older naming structure, how will they be affected?

Reply:
As far as I can see the filename is returned and no implicit convention is there for parsing the filename. I could be wrong though